Finalize some docs (#18)
herflis/concepts-temp: content/content-management.md

> For content management we mean the processes of collection, editing, organizing and delivering of any content.

Reviewer: ```suggestion
For content management we mean the processes of storing, editing, organizing and delivering content.
```

> What is a content in sensenet content management system?

Author: ```suggestion
# What is content regarding to sensenet?
```

In sensenet everything is content. Documents, tasks, users, projects, even system configuration files – everything – are stored and managed in the [content repository](/concepts/content-repository). It is the foundation that makes content management easy and powerful at the same time. All content items are organized in a [content tree](/concepts/content-tree), so you can always think in subtrees and inheritance when managing and presenting content.

# Basic content management actions in sensenet

> 

Reviewer: I would split these in two section:
- basic content management actions
- other additional actions

Preview, search, share are not basic actions in my opinion.

Author: and maybe they could be explained a bit more

- upload / download
- organize (structure)

To learn more go to [Actions](/concepts/actions)

> # Anybody can do anything?

Author: ```suggestion
# Can anybody do anything?
```

Author: I think the next few paragraphs should be rethought ⬇
